doc/train_parameters.md

```diff
@@ -0,0 +1,155 @@
+| shard            | train.shard           | integer       | 1             | distributed training if greater than 1. Fixed size of PS btw.                                                               |
+| shuffle          | train.shuffle         | bool/integer  | false         | bool: enable shuffle or not. integer: shuffle buffer size.                                                                  |
+| max_steps        | train.max_steps       | integer       | None          |                                                                                                                             |
+| eval steps       | eval.steps            | integer       | None          |                                                                                                                             |
+| start delay secs | eval.start_delay_secs | integer       | 120           |                                                                                                                             |
+| throttle_secs    | eval.throttle_secs    | integer       | 600           |                                                                                                                             |
+
+### COLUMN FOR
+The expressions in `COLUMN` contains the `Feature Columns` information. There could be multiple `COLUMN` blocks in the SQL. 
+
+The value of `FOR` keyword represents which parameter of the constructor method the `feature columns` assigning to.
+
+For example, the following SQL
+```sql
+...
+TRAIN
+	DNNLinearCombinedClassifier
+WITH
+	...
+COLUMN
+	NUMERIC(...) FOR linear_feature_columns
+COLUMN
+	BUCKET(...) FOR dnn_feature_columns
+...
+```
+will be translated to 
+```python
+	estimator = tf.estimator. DNNLinearCombinedClassifier(
+		linear_feature_columns = [tf.feature_column.numeric(...)],
+		dnn_feature_columns = [tf.feature_column.bucket(...)],
+		...
+	)
+```
+
+They were not only translated into [TensorFlow Feature Columns](https://www.tensorflow.org/guide/feature_columns), but also contains the encoding information of data such as `DENSE` or `SPARSE`. 
+
+For example, the following SQL
+```sql
+SELECT 
+...
+COLUMN
+	DENSE(c1, ...)
+	SPARSE(c2, ...)
+```
+represents that the `c1` field is the dense format and the `c2` field is the sparse format.
+
+<b>Here is a list of supported expression:</b>
+
+| expression | arguments of expression                                          | example                            |
+|------------|------------------------------------------------------------------|------------------------------------|
+| dense      | 1. field name (str) <br> 2. dense shape (integer) <br> 3. separator (str)  | dense(c1, 100, comma)              |
+| sparse     | 1. field name (str) <br> 2. sparse shape (integer) <br> 3. separator (str) | sparse(c2, 10000, comma)           |
```

Comment:
maybe add data samples to indicate the column type?

Reply:
It is in the `sqlflow/doc/alps_submitter.md`.